vendor/github.com/mongodb/mongo-tools-common/db/db.go

```diff
@@ -86,14 +87,14 @@ type ApplyOpsResponse struct {
 
 // Oplog represents a MongoDB oplog document.
 type Oplog struct {
-	Timestamp bson.MongoTimestamp `bson:"ts"`
+	Timestamp primitive.Timestamp `bson:"ts"`
```

Comment:
I'm a bit worried how this will impact mongodump. Did you test it?

Reply:
I did test it and it passes our current tests (integration and js). 

I had to make some slight [changes](https://github.com/mongodb/mongo-tools/blob/ac61328529fa098692aec9eb05123bc01156cf58/mongodump/oplog_dump.go) to mongodump as part of this, though. 

Mongo mirror uses its own oplog type, so it is unaffected by this change.